src/main/java/app/AppBuilder.java

```diff
@@ -97,7 +99,7 @@ public class AppBuilder {
 
     public AppBuilder() {
         cardPanel.setLayout(cardLayout);
-        cardPanel.setPreferredSize(new Dimension(600, 500));
+        cardPanel.setPreferredSize(new Dimension(WIDTH, HEIGHT));
```

Comment:
Thanks for creating the private static variables for the dimensions width and height, code looks alot cleaner now!